src/ocean_data_qc/fyskem/configs/consistency_check.yaml

```diff
@@ -0,0 +1,26 @@
+NTOT:
+  global: !!python/object:ocean_data_qc.fyskem.qc_checks.ConsistencyCheck
+    parameter_list: ['NTRA', 'NTRI','AMON']
+    upper_deviation: 0
+    lower_deviation: -1
+PTOT:
+  global: !!python/object:ocean_data_qc.fyskem.qc_checks.ConsistencyCheck
+    parameter_list: ['PHOS']
+    upper_deviation: 0
+    lower_deviation: -0.2
+# TOC:
+# 20% incertainty. TOC in mg/l, POC, DOC umol/l
+#   global: !!python/object:ocean_data_qc.fyskem.qc_checks.ConsistencyCheck
+#     parameter_list: ['POC', DOC]
+#     upper_deviation: 0
+#     lower_deviation: 0
```

Comment:
Checka inte in utkommenterad kod. Går utmärkt att stoppa i en issue istället.

Reply:
okejdå [lagt till här](https://github.com/nodc-sweden/ocean-data-qc/issues/67#issue-2602915750)